firefox_puppeteer/ui/windows.py

```diff
@@ -424,14 +429,21 @@ def navbar(self):
         See the :class:`~ui.navbar.NavBar` reference.
         """
 
-    @use_class_as_property('ui.tabbar.Tabs')
+    @property
```

Comment:
Why this change? This implementation is doing about the same thing as use_class_as_property.